lib/users/routes.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+/**
+ * @fileoverview Content system routing
+ */
+const express = require('express');
+const router = new express.Router();
+const config = require('config');
+const contentTypes = require('punchcard-content-types');
+const users = require('./');
+const multipart = require('connect-multiparty');
+const multipartMiddleware = multipart();
+const database = require('../database');
+
+/*
+Userss landing
+@function
+@name /users/
+*/
+router.get('/', (req, res) => {
+  let content;
+  database.select('*').from('users')
+  .then(rows => {
+    content = rows;
+  })
+  .catch(error => {
+    console.error(error);
+  })
+  .finally(() => {
+    res.render('users/landing', {
+      pageTitle: config.users.usersHome.title,
+      usersContent: content,
+      usersConfig: config.users,
+    });
+
+    return;
+  });
+});
+
+/*
+Users add-new page
+@function
+@name /users/add
+*/
+router.get('/add', (req, res) => {
+  return users.userContentType().then(userContentType => {
+    return contentTypes.form(userContentType[0])
+    .then(form => {
+      res.render('users/add', {
+        pageTitle: `${config.users.usersHome.title} :: Add`,
+        formContents: form.html,
+        validation: form.validation,
+      });
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  });
+});
+
+// POST method route
+router.post('/add', multipartMiddleware, () => {
+  console.log('NEW user added to database because awesome');
```

Comment:
@Snugug how do I get the data from the form? This is what `req.body` is returning:

```
{ '362978f8-a24d-4e09-8842-bb017da032d5': 'email@test.com',
  '4e95dbbd-10de-4da5-8b60-dfba3f127279': '111222',
  '2b1352c4-220c-4187-887f-7cc62cdb16ee': 'superuser' }
```

Is there a way for me to translate those ids into which input they represent in the current punchcard?